extensions/amp-story-interactive/0.1/amp-story-interactive-abstract.js

```diff
@@ -827,4 +828,32 @@ export class AmpStoryInteractive extends AMP.BaseElement {
       }
     });
   }
+
+  /**
+   * Reorders options data to account for scrambled or incomplete data.
+   *
+   * @private
+   */
+  orderData_() {
+    const numOptionElements = this.getOptionElements().length;
+    const orderedData = new Array(numOptionElements);
+    for (let i = 0; i < this.optionsData_.length; i++) {
+      const {index} = this.optionsData_[i];
+      if (index >= 0 && index < numOptionElements) {
+        orderedData[this.optionsData_[i].index] = this.optionsData_[i];
+      }
+    }
+
+    for (let i = 0; i < orderedData.length; i++) {
+      if (typeof orderedData[i] === 'undefined') {
```

Comment:
I think here you can just use `!orderedData[i]` as a boolean, since objects are _truthy_ and undefined/null is _falsey_ https://medium.com/coding-in-simple-english/what-are-truthy-values-in-javascript-e037bdfa76f8

Reply:
Will do, that's a lot simpler